auth-web/src/App.vue

```diff
@@ -32,7 +32,7 @@
           <v-icon>mdi-close</v-icon>
         </v-btn>
       </v-snackbar>
-      <navigation-bar :configuration="navigationBarConfig" :hide="!showNavigationBar" />
+      <BreadCrumb v-if="showNavigationBar" class="pl-8" :breadcrumbs="breadcrumbs" />
```

Comment:
Are you sure this needs "pl-8"? Or should it be in the same container as the body and only go to the container margins? Eg,

![image](https://user-images.githubusercontent.com/10002889/149220285-52b7f357-0cfa-4484-8e84-57052d841e91.png)

Reply:
Where it is right now, it does require it. I think there is additional padding inside the views, which required some additional padding on the app level. Can have another look though.

Reply:
The reason I'm asking is that the margins change depending on screen width.

The breacrumb should line up with the left side as above.

Reply:
For sure, this isn't achievable on the app level from what i can see even when placed inside the`app-body`. 
Would require to be about 4 levels down in the Views themselves.. so i'm not sure. Will keep looking.

Reply:
What if you wrap it in a container?

Or move it elsewhere?

Reply:
So this worked without issue in the other UI's and i think it's due to something locally overriding the `no-gutters` class that the shared component uses. 
Will keep looking in what the root cause of the override is, otherwise i will try to explicitly set. Don't want to change the Shared Component in this instance as i believe it is some local styling. 
👀

Reply:
OK yea, found it in the `layout` overrides.. basically nullifies any instance of `no-gutters` so i will look into the implications of remove this override.

Reply:
Removed the override and fixed some elements that were compensating for this previously